<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Camera_CatalogyClass</class>
 <widget class="QMainWindow" name="Camera_CatalogyClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>683</width>
    <height>289</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Каталог камер</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QTableWidget" name="tableWidget">
          <column>
           <property name="text">
            <string>Modul</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Price</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <layout class="QGridLayout" name="gridLayout">
        <item row="7" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Дата выпуска</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEdit_ModelName">
          <property name="locale">
           <locale language="English" country="UnitedStates"/>
          </property>
          <property name="maxLength">
           <number>30</number>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QSpinBox" name="spinBox_Price">
          <property name="minimum">
           <number>1000</number>
          </property>
          <property name="maximum">
           <number>150000</number>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Цена</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Разрешение матрицы</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Категория</string>
          </property>
         </widget>
        </item>
        <item row="7" column="2">
         <widget class="QPushButton" name="pushButton_delete">
          <property name="text">
           <string>Удалить</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QDoubleSpinBox" name="doubleSpinBox_CamMatrix">
          <property name="minimum">
           <double>2.000000000000000</double>
          </property>
          <property name="maximum">
           <double>20.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QCheckBox" name="checkBox_LensOptions">
          <property name="text">
           <string>Возможности смены объектива</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QSpinBox" name="spinBox_Weight">
          <property name="minimum">
           <number>100</number>
          </property>
          <property name="maximum">
           <number>8000</number>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Габариты</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Производитель</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QDateEdit" name="dateEdit"/>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="comboBox_Producer">
          <item>
           <property name="text">
            <string>Nicon</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Panasonic</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Sony</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Canon</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Olympus</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Зенит</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="6" column="2">
         <widget class="QPushButton" name="pushButton_save">
          <property name="text">
           <string>Сохранить</string>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QPushButton" name="pushButton_add">
          <property name="text">
           <string>Добваить</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Название модели</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QCheckBox" name="checkBox_IsDigital">
          <property name="text">
           <string>Цифровой</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="lineEdit_Dimensions">
          <property name="inputMask">
           <string/>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="maxLength">
           <number>8</number>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="comboBox_Category">
          <item>
           <property name="text">
            <string>Профессиональный</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Любительский</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Полупрофессиональный</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="5" column="2">
         <widget class="QPushButton" name="pushButton_add10">
          <property name="text">
           <string>Добавить 10</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Вес</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>683</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Camera_Catalogy.qrc"/>
 </resources>
 <connections/>
</ui>
